5.process_CP_features/scripts/0.merge_sc.py

```diff
@@ -80,47 +79,44 @@
-                    Image_FileName_561,
-                    Image_FileName_DNA
-
-
-
-                FROM
-                    read_parquet('per_image.parquet')
-                )
-            SELECT
-                *
-            FROM
-                Per_Image_Filtered AS per_image
-            LEFT JOIN read_parquet('per_cytoplasm.parquet') AS per_cytoplasm ON
-                per_cytoplasm.Metadata_ImageNumber = per_image.Metadata_ImageNumber
-            LEFT JOIN read_parquet('per_cells.parquet') AS per_cells ON
-                per_cells.Metadata_ImageNumber = per_cytoplasm.Metadata_ImageNumber
-                AND per_cells.Metadata_Cells_Number_Object_Number = per_cytoplasm.Metadata_Cytoplasm_Parent_Cells
-            LEFT JOIN read_parquet('per_nuclei.parquet') AS per_nuclei ON
-                per_nuclei.Metadata_ImageNumber = per_cytoplasm.Metadata_ImageNumber
-                AND per_nuclei.Metadata_Nuclei_Number_Object_Number = per_cytoplasm.Metadata_Cytoplasm_Parent_Nuclei
-                """,
-    },
+    # "run_20230920ChromaLiveTL_24hr4ch_MaxIP": {
+    #     "source_path": pathlib.Path(
+    #         f"{source_dir}/20230920ChromaLiveTL_24hr4ch_MaxIP/timelapse_4ch_analysis.sqlite"
+    #     ).resolve(),
+    #     "dest_path": pathlib.Path(
+    #         f"{output_dir}/20230920ChromaLiveTL_24hr4ch_MaxIP.parquet"
+    #     ).resolve(),
+    #     "preset": """WITH Per_Image_Filtered AS (
+    #             SELECT
+    #                 Metadata_ImageNumber,
+    #                 Image_Metadata_Well,
+    #                 Image_Metadata_FOV,
+    #                 Image_Metadata_Time,
+    #                 Image_PathName_488_1
+    #                 Image_PathName_488_2,
+    #                 Image_PathName_561,
+    #                 Image_PathName_DNA,
+    #                 Image_FileName_488_1,
+    #                 Image_FileName_488_2,
+    #                 Image_FileName_561,
+    #                 Image_FileName_DNA
+    #             FROM
+    #                 read_parquet('per_image.parquet')
+    #             )
+    #         SELECT
+    #             *
+    #         FROM
+    #             Per_Image_Filtered AS per_image
+    #         LEFT JOIN read_parquet('per_cytoplasm.parquet') AS per_cytoplasm ON
+    #             per_cytoplasm.Metadata_ImageNumber = per_image.Metadata_ImageNumber
+    #         LEFT JOIN read_parquet('per_cells.parquet') AS per_cells ON
+    #             per_cells.Metadata_ImageNumber = per_cytoplasm.Metadata_ImageNumber
+    #             AND per_cells.Metadata_Cells_Number_Object_Number = per_cytoplasm.Metadata_Cytoplasm_Parent_Cells
+    #         LEFT JOIN read_parquet('per_nuclei.parquet') AS per_nuclei ON
+    #             per_nuclei.Metadata_ImageNumber = per_cytoplasm.Metadata_ImageNumber
+    #             AND per_nuclei.Metadata_Nuclei_Number_Object_Number = per_cytoplasm.Metadata_Cytoplasm_Parent_Nuclei
+    #             """,
+    # },
```

Comment:
Recommend removing commented out code if not used.

Reply:
removed